src/plone/app/mosaic/profiles/upgrades/to_6003/registry.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0"?>
+<registry>
+
+  <records interface="plone.app.mosaic.interfaces.ITile"
+           prefix="plone.app.mosaic.app_tiles.plone_app_standardtiles_discussion"
+  >
+    <value key="name">plone.app.standardtiles.discussion</value>
+    <value key="label">Discussion</value>
+    <value key="category">applications</value>
+    <value key="tile_type">app</value>
+    <value key="default_value" />
+    <value key="read_only">false</value>
+    <value key="settings">false</value>
```

Comment:
This would reset all custom integrator values to default values on update. I'd recommend to only set the `<value key="settings">false</value>` in the upgrade profile.

Reply:
@petschki Thanks for pointing this out! I updated the upgrade profile to only change the relevant value